src/foam/core/Static.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0 
+ */
+
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Static',
+  extends: 'foam.core.AbstractMethod',
+
+  documentation: 'An Axiom for defining static methods.',
+
+  properties: [
+    //get prop from foam.core.AbstractMethod
+  ],
+
+  methods: [
+    function exportAs(obj) {
+    },
+    function isStatic(){ return true;},
+    function installInClass(cls) {
+      Object.defineProperty(
+        cls,
+        foam.String.constantize(this.name),

```

Comment:
I don't think we should call constantize() in this case. Just use the name the developer provides.